src/components/swap/PendingModalContent/ErrorModalContent.tsx

```diff
@@ -0,0 +1,73 @@
+import { Trans } from '@lingui/macro'
+import { ButtonPrimary } from 'components/Button'
+import { ColumnCenter } from 'components/Column'
+import QuestionHelper from 'components/QuestionHelper'
+import Row from 'components/Row'
+import { AlertTriangle } from 'react-feather'
+import { useTheme } from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+import { PendingModalContainer } from '.'
+
+export enum PendingModalError {
```

Comment:
this whole file is just copied from the previous `src/components/swap/PendingModalContent.tsx`